.circleci/config.yml

```diff
@@ -116,112 +117,46 @@ jobs:
           region: $ECR_REGION
       - run: |
           aws ecr get-login-password --region $ECR_REGION | docker login --username AWS --password-stdin ${AWS_ECR_REGISTRY_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com
```

Comment:
My only comment on this is that we don't use ECR in general, we use quay.io. However, if this works using ECR for now, then we can get this done, then move over to quay.io?